ios/RNHTMLtoPDF/RNHTMLtoPDF.m

```diff
@@ -170,6 +170,12 @@ - (instancetype)init
         _paddingLeft = [RCTConvert float:options[@"padding"]];
         _paddingRight = [RCTConvert float:options[@"padding"]];
     }
+    
+    if (@available(iOS 16.4, *)) {
+      if (options[@"backgroundGraphics"]) {
+        _webView.configuration.preferences.shouldPrintBackgrounds = [options[@"backgroundGraphics"] boolValue];
```

Comment:
I believe the option should be converted via `RCTConvert`.

```suggestion
        _webView.configuration.preferences.shouldPrintBackgrounds = [RCTConvert BOOL:options[@"backgroundGraphics"]];
```

Reply:
Also, `[WKWebView configuration]` should only be accessed from the main thread.

```suggestion
        dispatch_sync(dispatch_get_main_queue(), ^{
            _webView.configuration.preferences.shouldPrintBackgrounds = [RCTConvert BOOL:options[@"backgroundGraphics"]];
        });
```

Reply:
Thank you for saving my day 🥳